content/Lab1/50_Firewall_Configuration/index.md

```diff
@@ -1,12 +1,12 @@
 ---
 title: "Firewall Configuration - Programmatically"
-weight: 70
+weight: 50
 ---
 
 In this section, we will perform and firewall registration and configuration using Terraform. 
 We will first walk through the firewall configuration prerequisites and then proceed with the firewall configuration deployment.
 
 
-[Firewall Configuration - Prerequisites](../70_Firewall_Configuration/1_Prerequisites.md)
+[Firewall Configuration - Prerequisites](../50_Firewall_Configuration/1_Prerequisites.md)
 
-[Firewall Configuration - Deployment](../70_Firewall_Configuration/2_Firewall_Config.md)
\ No newline at end of file
+[Firewall Configuration - Deployment](../50_Firewall_Configuration/2_Firewall_Config.md)
```

Comment:
please rename the respective md titles so that it matches with this text e.g. 1_network_resources.md title should be AWS Network Resources - Description